lib/dialects/abstract/query.js

```diff
@@ -514,7 +514,19 @@ AbstractQuery.prototype.isUpdateQuery = function() {
 
 AbstractQuery.prototype.handleSelectQuery = function(results) {
   var result = null;
-
+  // Map raw fields to arbitrary names if a mapping is provided
+  if (this.options.fieldMap) {
```

Comment:
Hmm, it looks like `this.options.raw` isn't set if a model is passed in [query.js#L671](https://github.com/sequelize/sequelize/blob/master/lib/sequelize.js#L671). Is there a way to know if the query originated from user input on `sequelize.query` directly without using `options.raw`?